### PR TITLE
Avoid ambiguity as of the duration of 'contracts'

### LINKED ISSUE #902: Expand tender.contractPeriod description
> The period over which the contract is estimated or required to be active. If the tender does not specify explicit dates, the duration field may be used.

The EU standard procurement forms use the construction "Duration of the contract, framework agreement or dynamic purchasing system", to avoid ambiguity around which contract this period refers to (as there can be many contracts over different periods, especially in the latter cases).

### PR DESCRIPTION
**Pull request by ColinMaudry**
Closes #902

### CODE REVIEW

**Comment by ColinMaudry** [comment]
@jpmckinney Do you know what's wrong here?

I ran `python util/make_dereferenced_release_schema.py` and merged the 1.2-dev branch in this one.

**Comment by jpmckinney** [comment]
I think the search backend was under high load due to the many PRs, so I'll just re-run the checks.

File: schema/release-schema.json [hunk 1]
```diff
@@ -389,7 +389,7 @@
           "$ref": "#/definitions/Period"
         },
         "contractPeriod": {
-          "description": "The period over which the contract is estimated or required to be active. If the tender does not specify explicit dates, the duration field may be used.",
+          "description": "The period over which the contract, the framework agreement or the dynamic purchasing system is estimated or required to be active. If the tender does not specify explicit dates, the duration field may be used.",
```
Comment (jpmckinney) [inline schema/release-schema.json]:
```suggestion
          "description": "The period over which the contract, framework agreement or dynamic purchasing system is estimated or required to be active. If the tender does not specify explicit dates, the duration field may be used.",
```

**Comment by jpmckinney** [comment]
Closing in favor of the approach described in https://github.com/open-contracting/standard/issues/827#issuecomment-728036400, whereby all mentions of "contract" would defer to #896, rather than repeating the long statement everywhere.

**Pull request closed without merging**